appender/src/main/java/com/vlkan/log4j2/redis/appender/RedisAppender.java

```diff
@@ -296,6 +324,12 @@ public static Builder newBuilder() {
         @PluginBuilderAttribute
         private boolean debugEnabled = false;
 
+        @PluginBuilderAttribute
+        private String sentinelNodes;
+
+        @PluginBuilderAttribute
+        private String sentinelMaster;
```

Comment:
Would you mind adding these two new parameters into the table of `RedisAppender` parameters in `README.md`, please? While you are at it, add yourself to the list of contributors in `README.md` too. :wink: